packages/interactivity-router/src/index.js

```diff
@@ -81,7 +78,18 @@ pages.set(
 	Promise.resolve( regionsToVdom( document ) )
 );
 
+// Variable to store the current navigation.
+let navigatingTo = '';
+
 export const { state, actions } = store( 'core/router', {
+	state: {
+		url: '',
```

Comment:
We need to populate this with the current url.

Reply:
Fixed in https://github.com/WordPress/gutenberg/pull/58377/commits/f207a7998034bf15c012202e9137111d75e17adc.